nodes/src/nodes/llm_ibm_watson/ibm_watson.py

```
@@ -22,7 +22,7 @@
 # =============================================================================
 
 import re
-from typing import Any, Dict
+from typing import Any, Callable, Dict
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)